src/app/card/card.component.view.spec.ts

```diff
@@ -124,7 +161,7 @@ describe('MzCardComponent:view', () => {
         <mz-card [textClass]="'class-x'"></mz-card>`).then((fixture) => {
 
         nativeElement = fixture.nativeElement;
```

Comment:
This line seem to appear a whole lot of time but nativeElement never actually used. Can we remove it?